lasagne/utils.py

```diff
@@ -5,17 +5,40 @@
 
 
 def floatX(arr):
-    """
-    Shortcut to turn a numpy array into an array with the
-    correct dtype for Theano.
+    """Converts numpy array to one with the correct Theano data-type.
```

Comment:
I think the phrase "Theano data-type" might confuse a few people, since we're talking about numpy dtypes here (not Theano tensors or shared variables). Refering to this as 'dtype' is probably clearer.